spark/common/src/main/java/com/netease/arctic/spark/utils/MixedFormatSparkUtil.java

```diff
@@ -0,0 +1,26 @@
+package com.netease.arctic.spark.utils;
+
+import com.netease.arctic.ams.api.TableFormat;
+import com.netease.arctic.hive.HiveTableProperties;
+import com.netease.arctic.hive.utils.CompatibleHivePropertyUtil;
+import com.netease.arctic.table.TableProperties;
+import org.apache.spark.sql.connector.catalog.Table;
+
+public class MixedFormatSparkUtil {
+
+  /**
+   * check a Spark is a mixed-format table.
+   * @param table the spark table loaded from hive catalog
+   * @return is it a mixed format table.
+   */
+  public static boolean isMixedFormatTable(Table table) {
```

Comment:
Do you think it's better to use `table properties` or `instance of` to determine the table format?

We have implemented similar methods in core/hive module, using `instance of`, where
```
com.netease.arctic.utils.TableTypeUtil#isIcebergTableFormat
com.netease.arctic.hive.utils.TableTypeUtil#isHive
```

I think we should also put `isMixedTableFormat` to `TableTypeUtil`.

Reply:
Currently, TableTypeUtil determines the instance type based on ArcticTable. Since this logic needs to be determined before loading the table, it is best to avoid directly depending on the ArcticTable object.

In addition, the current MixedFormat is being refactored, and the judgment based on properties is not applicable to all catalogs, so the code is temporarily implemented in the Spark module. In the future, when the mixed-format refactoring is completed, a unified TableTypeUtil entry will be provided in the core module.